OpenDreamRuntime/Procs/Native/DreamProcNativeDatum.cs

```diff
@@ -0,0 +1,45 @@
+using OpenDreamRuntime.Objects;
+using OpenDreamRuntime.Objects.Types;
+
+namespace OpenDreamRuntime.Procs.Native;
+
+internal static class DreamProcNativeDatum {
+    [DreamProc("Write")]
+    [DreamProcParameter("F", Type = DreamValue.DreamValueTypeFlag.DreamObject)]
+    public static DreamValue NativeProc_Write(NativeProc.Bundle bundle, DreamObject? src, DreamObject? usr) {
+        var probablySavefile = bundle.GetArgument(0, "F");
```

Comment:
Insane though it seems, I think this should actually be done in the savefile's `OperatorIndexAssign` and the native `Write()` and `Read()` should be stubs. The behaviour is that the writing/reading happens regardless of whether the native `Write`/`Read` are called, and you cannot cancel it by, for example, returning early.

To be clear, you could do this:
```
/obj/test
  Write(savefile/F)
    if(some_condition)
      ..()
    else
      return null
```

and it would still do the write even if `some_condition` was true